### PR TITLE
{chem}[intel/2015b] DIRAC 14.1 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
requires https://github.com/hpcugent/easybuild-easyblocks/pull/778

### CODE REVIEW

File: easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+dependencies = [
+     ('zlib', '1.2.8'),
+     ('Boost', '1.54.0', versionsuffix),
+]
+builddependencies = [('CMake', '3.4.1')]
+
+separate_build_dir = True
+
+configopts = "-DENABLE_MPI=ON -DCMAKE_BUILD_TYPE=release"
+prebuildopts = "unset LDFLAGS && "
```
Comment (wpoely86) [inline easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb]: add a comment why this is unset

File: easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+preinstallopts = prebuildopts
+
+# see http://diracprogram.org/doc/release-14/installation/testing.html
+#runtest = 'test'
```
Comment (wpoely86) [inline easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb]: they don't work out of the box?

File: easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb]: I'll need to resort to an easyblock to get the tests running, I'll document it there

File: easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb]: they take forever, and require env vars to be set, see http://diracprogram.org/doc/release-14/installation/testing.html

Comment (wpoely86) [inline easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb]: even `-L short`?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 6 out of 11 (2 easyconfigs in this PR)
Linux debian 8.1, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.9
See https://gist.github.com/d9c64693b09a218b0482 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5416c6b110dc378f6ac5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ff92bd96e2c0a5122788 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/fe2296c2326a9745df41 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Easyblock is merged, so good to go now.

Thanks for the review @wpoely86!

**Pull request merged**